README.md

```diff
@@ -1,2 +1,32 @@
 # My-Portfolio
+
 Hello World, I am a full stack web developer in the making.
+
+Built With 
+HTML 
+CSS 
+Visual Studio
+
+Getting Started 
+
+This is our second project in the Full Stack program where we must utilize the skills we have learned in the previous lessons and design the sekelton of our portfolio website that will be used when we start to seek employment. This website will have all our projects that we have worked on.
+
+Authors 
+👤 Author1
+
+GitHub: TechnoAndy 
+Twitter: @Adiegaardt 
+LinkedIn: http://www.linkedin.com/in/andrea-diegaardt-110203116
+
+🤝 Contributing 
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+Show your support Give a ⭐️ if you like this project!
+
+Acknowledgments 
+
+A shout out to my team who were my motivation.
+
+📝 License This project is MIT licensed.
```

Comment:
- [OPTIONAL] Try using `#` for titles in your markdown file. This will make your README looks more elegant. Also try providing steps about how to use your app, how to clone the repo, etc.

Reply:
Change has been made as requested. Thank you.